ana-febles.txt

```diff
@@ -0,0 +1,145 @@
+https://codepen.io/AnaFebles/pen/vYLrexy
+
+
+1. What is the difference between an inline element and a block element? 
+in line: it will not force a new line break, everything will be aling in the same line. we cannot put in block elements on in line ones. 
+block: will forcve a new line to be created.
+
+2. What happens when an element is positioned absolutely?
+it is a fixed psititon, basically it will not move from that place.
+
+3. How do I make an element take up only the amount of space it needs but also have the ability to give it a width? 
+div
+
+4. Name 3 elements that are diplay block by default, 2 elements that are display inline by default and 1 element that is display inline-block by default 
+
+block: div, header, section
+in line: span, img
```

Comment:
The 3 block elements you named are correct, however `img` is [not `inline` by default](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Img#Styling_with_CSS).